docs/source/basics/preference_datasets.rst

```diff
@@ -0,0 +1,162 @@
+.. _preference_dataset_usage_label:
+
+===================
+Preference Datasets
+===================
+
+
+Preference datasets are used for reward modelling, where the downstream task is to fine-tune a base model
+to capture some underlying human preferences. Currently, these datasets are used in torchtune with the
+Direct Preference Optimization (DPO) `recipe <https://github.com/pytorch/torchtune/blob/main/recipes/lora_dpo_single_device.py>`_.
+
+The ground-truth in preference datasets is usually the outcome of a binary comparison between two completions for the same prompt,
+and where a human annotator has indicated that one completion is more preferable than the other, according to some pre-set criterion.
+These prompt-completion pairs could be instruct style (single-turn, optionally with a single prompt), chat style (multi-turn), or
+some other set of interactions between a user and model (e.g. free-form text completion).
```

Comment:
is this true, or do we only support preference chat? I guess as long as you make the transform it should work for all three, but your example below implies only chat

Reply:
Since we support an optional system prompt it should work for all three right? That was the whole point of the preference dataset refactor to support arbitrary interactions